modules/n1ql/pages/n1ql-language-reference/insert.adoc

```diff
@@ -1155,7 +1163,7 @@ For example, the following REST API call throws an error because of spaces befor
 
 [source,console]
 ----
-$ curl -v http://localhost:8093/query/service -d 'prepared="ins_india"&$key="airport_10002" & $faa_code="BLR"'
+$ curl -v http://localhost:8093/query/service -u Administrator:password -d 'prepared="ins_india"&$key="airport_10002" & $faa_code="BLR"'
```

Comment:
I've not been able to reproduce the error we're expecting here when I run the curl command, so not 100% how to cover this. The request  seems to succeed for me even with the spacing before the '&'.

Reply:
Yep, works fine for me too. Delete that entire Note block.